src/server/compositor/default_configuration.cpp

```diff
@@ -33,16 +32,6 @@ namespace mc = mir::compositor;
 namespace ms = mir::scene;
 namespace mg = mir::graphics;
 
-std::shared_ptr<ms::BufferStreamFactory>
-mir::DefaultServerConfiguration::the_buffer_stream_factory()
```

Comment:
Do we need to do anything in the `symbols.map` file to remove this?

Reply:
https://github.com/canonical/mir/actions/runs/8920919391/job/24499927929?pr=3368 :)

Reply:
:tada: 

Hurray for extra CI checks!